demo/flutter/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,31 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="ai.picovoice.flutter.leoparddemo">
+
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.RECORD_AUDIO"/>
+
+    <application
+        android:label="Leopard Demo"
+        android:icon="@mipmap/ic_launcher"
+        android:roundIcon="@mipmap/ic_launcher_round"   
+        android:allowBackup="false"
+        android:theme="@style/NormalTheme">
+        <activity
+            android:name=".MainActivity"
+            android:label="@string/app_name"
```

Comment:
```suggestion
            android:exported="true"
            android:label="@string/app_name"
```
I needed this in order to build and run on my device -> targetting Android SDK 31